src/css/base.scss

```diff
@@ -2107,14 +2108,19 @@ textarea,
     width: 303px;
 }
 
-.w-350 {
-    width: 350px;
+.w-320 {
+    width: 320px !important;
+}
+.text-align-left{
+    text-align: left !important;
```

Comment:
I think, we already have this prop. Can you please check and update accordingly?